app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: ["Distance", int, float]) -> object:
```

Comment:
wrong type annotation. You can use  Union[ ] or `|` symbol to indicate  that a type could be one of multiple options

and the return type annotation should be more specific